src/app/item-detail/item-detail.component.ts

```diff
@@ -138,8 +115,19 @@ export class ItemDetailComponent implements OnInit {
             }
```

Comment:
(λίγο παραπάνω) Γιατί κάνεις `this.stars.map` χωρίς να χρησιμοποιείς το array που δημιουργείται; Μάλλον `forEach` ήθελες. Επίσης δεν μου αρέσει η λογική του πώς το έχεις κάνει. Φαντάζομαι ακόμα κι εσύ δεν θυμάσαι τι κάνεις εδώ :Ρ Δες μήπως γίνεται πιο απλά.